### PR TITLE
hover and show

### PR DESCRIPTION
**Pull request by jordansread**
@mbucknell Is this what you are looking for on hover? 

also, @jiwalker-usgs not sure how we are dealing w/ play and pause functionality, but I mocked it up in a way I thought might be a start

### CODE REVIEW

**Comment by mbucknell** [comment]
Yep that's what I need.

**Comment by jordansread** [comment]
this is what we have to work w/ if you build this version:
image

I am now going to start getting those in the right place and add axes labels, etc

File: scripts/process/process_bar_chart.R [hunk 1]
```diff
@@ -20,7 +20,9 @@ process.bar_chart <- function(viz){
     xml_add_child(g.bars, 'rect', 
                   x = as.character((i-1)*(bin.w+spc)), 
                   height = as.character(bin.h[i]), 
-                  width = as.character(bin.w), y = as.character(h - bin.h[i]), id = paste0('yr', bars$year[i]))
+                  width = as.character(bin.w), y = as.character(h - bin.h[i]), id = paste0('yr', bars$year[i]), 
+                  onmousemove = sprintf("hovertext('%s gages');vizlab.showyear('%s')", bars$n[i], bars$year[i]),
```
Comment (jiwalker-usgs) [inline scripts/process/process_bar_chart.R:24]: @mbucknell should we namespace the hovertext function `vizlab.hovertext`?

Comment (mbucknell) [reply scripts/process/process_bar_chart.R:24]: No. The way I've coded this each viz would still create the "hovertext" handler. There may be more that you do in the handler besides just showing/hiding the hover text. In water use, there is code to send google events for instance.

Comment (jiwalker-usgs) [reply scripts/process/process_bar_chart.R:24]: ok, makes sense.

**Pull request merged**